src/Evolution/Systems/Cce/AnalyticSolutions/BouncingBlackHole.hpp

```diff
@@ -69,9 +69,8 @@ struct BouncingBlackHole : public WorldtubeData {
 
   explicit BouncingBlackHole(CkMigrateMessage* /*unused*/) noexcept {}
 
-
   // clang doesn't manage to use = default correctly in this case
-  // NOLINTNEXTLINE(hicpp-use-equals-default)
+  // NOLINTNEXTLINE(hicpp-use-equals-default,modernize-use-equals-default)
```

Comment:
Did you check if the first warning still applies? I've noticed some cases where the newer (llvm 10) clang-tidy has started flagged the same line of code under a new warning.